tests/tr1/tests/random5/test.cpp

```diff
@@ -426,7 +426,7 @@ static void tdiscrete() {
 
     STD vector<double> vec(4, 1.0);
 
-    dist_t dist1(STD initializer_list<double>(vec.data(), vec.data() + vec.size()));
+    dist_t dist1(vec.begin(), vec.end());

```

Comment:
I'm concerned that this is a behavioral change from using `discrete_distribution<>`'s `initializer_list<double>` constructor to the range constructor - for all I know this could be the only coverage we have of that constructor. Could we use:
```suggestion
    dist_t dist1{1.0, 1.0, 1.0, 1.0};
```
here instead?